sky/utils/controller_utils.py

```diff
@@ -754,15 +776,13 @@ def maybe_translate_local_file_mounts_and_sync_up(task: 'task_lib.Task',
         if os.path.isfile(os.path.abspath(os.path.expanduser(src))):
             copy_mounts_with_file_in_src[dst] = src
             continue
-        bucket_name = constants.FILE_MOUNTS_BUCKET_NAME.format(
-            username=common_utils.get_cleaned_username(),
-            id=f'{run_id}-{i}',
-        )
         new_storage_mounts[dst] = storage_lib.Storage.from_yaml_config({
             'name': bucket_name,
             'source': src,
             'persistent': False,
             'mode': 'COPY',
+            'store': store,
+            'bucket_sub_path': f'job-{run_id}/local-file-mounts/{i}',
```

Comment:
Same as above:
> Rename this to `_bucket_sub_path`? Let's start with keeping this arg hidden and for internal use only. We can expose it publicly later if needed.

Reply:
```
sky/data/storage.py:259:17: C0103: Argument name "_bucket_sub_path" doesn't conform to '^[a-z][a-z0-9_]*$' pattern (invalid-name)
```
I can make it into a config variable `_bucket_sub_path` and the private member `self._bucket_sub_path`, but function arguments cannot start with an `_` prefix.